platform/barefoot/docker-syncd-bfn-rpc.mk

```diff
@@ -9,7 +9,7 @@ $(DOCKER_SYNCD_BFN_RPC)_DEPENDS += $(SYNCD_RPC_DBG) \
                                    $(LIBSAIMETADATA_DBG) \
                                    $(LIBSAIREDIS_DBG)
 endif
-$(DOCKER_SYNCD_BFN_RPC)_LOAD_DOCKERS += $(DOCKER_SYNCD_BFN)
+$(DOCKER_SYNCD_BFN_RPC)_LOAD_DOCKERS += $(DOCKER_SYNCD_BASE)
```

Comment:
why it is base? rpc docker is based on syncd.

Reply:
DOCKER_SYNCD_BASE expands to docker-syncd-$(DOCKER_SYNCD_PLATFORM_CODE).gz,  which in this case would be docker-syncd-bfn.gz